asdf/tags/wcs/wcs.py

```diff
@@ -295,3 +314,47 @@ def assert_equal(cls, old, new):
         assert old.name == new.name
         for old_frame, new_frame in zip(old.frames, new.frames):
             helpers.assert_tree_match(old_frame, new_frame)
+
```

Comment:
I may be missing something in a previous email, but why does ICRS get special treatment for frames here? None of the others have an explicit class definition as far as I understand.